### ISSUE TITLE
Create profile UI

### ISSUE THREAD

**Issue by jlmonge** [jlmonge/4nderground#4]
_The Profile is the UI of the Account._

A Profile Page consists of:
(not your account)
- [x] External links to redirect interested listeners.
- ~~Ability to ignore (you can't see their posts, but they can see yours).~~ *Postponed until the more relevant account management update.*
- ~~Ability to block (you can't see their posts, and they can't see yours).~~ *Postponed until the more relevant account management update.*

(your account)
- [x] External links to redirect interested listeners, plus an edit button to enter edit mode.
- [x] Logout button
- [x] A button to go to the settings page with the following (non-functional):
  - Change email
  - Change password
  - Delete account
  - See ignored users & option to unignore
  - See blocked users & option to unblock

General
- [x] In Navbar, replace Logout button with Avatar.
- [x] A profile page must be a modal.
- [x] Clicking an Avatar must bring up the profile page.

The following are features that are to be done later, just not in this initial implementation.
- Report account
- Fill in the settings/ignore/block skeleton (functionality)

The following are features that are NOT planned to be added in the foreseeable future, along with reasoning why.
- Following system: This is not a popularity content.

**Comment by jlmonge** [comment]
shh
